modules/nodes-cluster-limit-ranges-viewing.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * nodes/cluster/limit-ranges.adoc
+
+[id="nodes-cluster-limit-viewing_{context}"]
+= Viewing a Limit Range
```

Comment:
So as it happens, this is an object: _LimitRange_:

https://docs.openshift.com/container-platform/4.5/rest_api/schedule_and_quota_apis/limitrange-core-v1.html

Reply:
@jboxman The intro of the topic states: _A limit range, defined by a LimitRange object_.
That is the format I tried to use. A LimitRange object configures a limit range.

Reply:
The user, I would think, wants to know what the limits are. Not to examine an object. WDYT?